.travis.yml

```diff
@@ -8,6 +8,7 @@ cache:
   - node_modules
 node_js:
   - 'stable'
+  - '8.12.0'
```

Comment:
this node version doesn't appear to be available for Unix so tests are failing when try to run against this version in TravisCI